src/hooks/useOrders.ts

```diff
@@ -118,5 +146,12 @@ export function useOrders(): Result {
     dispatch(overwriteOrders([]))
   }, [userAddress, networkId, forceOrdersRefresh, dispatch])
 
-  return { orders, isLoading, forceOrdersRefresh }
+  return {
+    orders,
+    isLoading,
+    forceOrdersRefresh,
+    get pendingOrders(): AuctionElement[] {
+      return networkId && userAddress ? pendingOrders[networkId][userAddress] : emptyArray
```

Comment:
This was the problem with rerender?
Stable reference will fix this :+1:

Reply:
yeah all this has changed locally, ill update now